pkg/distro/BUILD

```diff
@@ -36,8 +36,8 @@ pkg_tar(
     srcs = [
         ":small_workspace",
         "//:standard_package",
-        "//toolchains:standard_package",
         "//releasing:standard_package",
+        "//toolchains:standard_package",
```

Comment:
Consider mentioning buildifier updates in your final message.  I see it in one of your intermediate commits.

Reply:
Will do.